ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/__Snapshots__/StoreKitIntegrationTests/SK2-testCanPurchasePackage.1.json

```diff
@@ -15,7 +15,7 @@
           "isActive" : true,
           "isSandbox" : true,
           "ownershipType" : "PURCHASED",
-          "periodType" : "INTRO",
+          "periodType" : "TRIAL",
```

Comment:
This changed because we were sending up SK1 receipts in iOS v4.X with SK2, and with iOS v5.X we're sending up SK2 JWTs, which treat this as a trial instead of intro